tests/test_tga.py

```diff
@@ -190,17 +190,17 @@ def test_token_level_loss_logging(self):
         gold_data = {
             'beam': {
                 'text_token_info': [
-                    ('__start__', 0.0, 1.0),
-                    ('5', -2.5510462364763953e-05, 0.0),
-                    ('__end__', -1.1920922133867862e-06, 0.0),
+                    ('__start__', {"token_prob": 0.0, "token_rank": 1}),
+                    ('5', {"token_prob": 0.999, "token_rank": 0}),
```

Comment:
i'd kinda prefer to keep them in logprobs if possible

Reply:
@emilydinan wanted to do probs not logprobs. Either is good for me though

Reply:
Logprobs are significantly more accurate...

Reply:
Logprobs are fine but we were using un-normalized logprobs before. So if we want to use logprobs it will be better to normalize them first

Reply:
Okay, I'll do normalized logprobs then!